dissect/target/plugins/apps/browser/chromium.py

```diff
@@ -371,6 +378,119 @@ def extensions(self, browser_name: Optional[str] = None) -> Iterator[BrowserExte
                 except (AttributeError, KeyError) as e:
                     self.target.log.info("No browser extensions found in: %s", json_file, exc_info=e)
 
+    def _get_local_state_key(self, local_state_path: TargetPath, chromium_passwords: list) -> Union[bytes, bool]:
+        """Get the Chromium os_crypt encrypted_key and decrypt the key using DPAPI."""
+
+        if not local_state_path.exists():
+            self.target.log.warning(f"File {local_state_path} does not exist.")
+            return False
+
+        try:
+            local_state_conf = json.loads(local_state_path.read_text())
+        except json.JSONDecodeError:
+            self.target.log.warning(f"File {local_state_path} does not contain valid JSON.")
+            return False
+
+        if "os_crypt" not in local_state_conf:
+            self.target.log.warning(
+                f"File {local_state_path} does not contain os_crypt, Chrome is likely older than v80."
+            )
+            return False
+
+        encrypted_key = base64.b64decode(local_state_conf["os_crypt"]["encrypted_key"])[5:]
+        decrypted_key = self.target.dpapi.decrypt_blob(encrypted_key, chromium_passwords)
+        return decrypted_key
+
+    def passwords(self, browser_name: str = None, chromium_passwords: list = None) -> Iterator[BrowserPasswordRecord]:
+        """Return browser password records from Chromium browsers.
+
+        Chromium on Linux has ``basic``, ``gnome`` and ``kwallet`` methods for password storage:
+            - ``basic`` ciphertext prefixed with ``v10`` and encrypted with hard coded parameters.
+            - ``gnome`` and ``kwallet`` ciphertext prefixed with ``v11`` which is not implemented (yet).
+
+        Chromium on Windows uses DPAPI user encryption. Passwords from Chromium before and after version 80 can be
+        decrypted. The SHA1 hash of the user's password or the plaintext password is required to decrypt passwords.
+
+        You can supply a SHA1 hash or plaintext password using the ``--passwords`` option.
+
+        Resources:
+            - https://chromium.googlesource.com/chromium/src/+/master/docs/linux/password_storage.md
+            - https://chromium.googlesource.com/chromium/src/+/master/components/os_crypt/sync/os_crypt_linux.cc#40
+        """
+
+        for user, db_file, db in self._iter_db("Login Data"):
+            rows = db.table("logins").rows()
+            decrypted_key = False
+
+            if self.target.os == OperatingSystem.WINDOWS.value:
+                local_state_path = db_file.parent.parent.joinpath("Local State")
+                decrypted_key = self._get_local_state_key(local_state_path, chromium_passwords)
+
+            for row in rows:
+                encrypted_password: bytes = row.password_value
+                decrypted_password = ""
```

Comment:
```suggestion
                decrypted_password = None
```
`None` is a better indicator of a "failed to decrypt" state.